src/include/daos_srv/bio.h

```diff
@@ -58,6 +58,11 @@ struct bio_iov {
 	/* Data length in bytes */
 	size_t		 bi_data_len;
 	bio_addr_t	 bi_addr;
+	/** can be used to fetch more than actual address. Useful if more
+	 * data is needed for processing (like checksums) than requested
+	 */
+	size_t		 bi_extra_begin; /** bytes before */
```

Comment:
minor nit...prefix_len and suffix_len?   Can you add a comment on why there is both extra_begin and extra_end?   Why not one or the other?

Reply:
thanks, those are way better. I'll update the comment. Let me know what you think.